prow/config/config.yaml

```diff
@@ -221,6 +223,22 @@ presubmits:
           value: "TRUE"
         image: quay.io/metal3-io/capbm-unit
```

Comment:
Can you update this to use the `:master` tag?  Once we have a `v1alpha2` tag, quay.io won't automatically advance `latest`, since it doesn't know which branch to follow